From pinglinux@gmail.com  Fri Jan 15 11:03:03 2010
From: Ping <pinglinux@gmail.com>
Date: Thu, 7 Jan 2010 16:49:59 -0800
Subject: serial/8250_pnp: Add a new Fujitsu Wacom Tablet PC device
To: Andrew Morton <akpm@linux-foundation.org>
Cc: alan@lxorguk.ukuu.org.uk, greg@kroah.com, jkosina@suse.cz,  mjg59@srcf.ucam.org, mjg@redhat.com
Message-ID: <167e8a331001071649l3d8df335w2b2422984bb245e8@mail.gmail.com>

From: Ping Cheng <pingc@wacom.com>

This is a new two finger touch Fujitsu Wacom Tablet PC

Signed-off-by: Ping Cheng <pingc@wacom.com>
Cc: stable <stable@kernel.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/serial/8250_pnp.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/serial/8250_pnp.c
+++ b/drivers/serial/8250_pnp.c
@@ -346,6 +346,8 @@ static const struct pnp_device_id pnp_de
 	{	"FUJ02E5",		0	},
 	/* Fujitsu P-series tablet PC device */
 	{	"FUJ02E6",		0	},
+	/* Fujitsu Wacom 2FGT Tablet PC device */
+	{	"FUJ02E7",		0	},
 	/*
 	 * LG C1 EXPRESS DUAL (C1-PB11A3) touch screen (actually a FUJ02E6 in
 	 * disguise)
